compiler/one-cmds/one-infer

```diff
@@ -206,6 +209,16 @@ def main():
     # run backend driver
     _utils._run(infer_cmd, err_prefix=ntpath.basename(driver_path))
 
+    # run post process script if it's given
+    if _utils._is_valid_attr(args, 'post_process'):
+        # Currently, it is only supported to use one-venv.
+        # TODO: support local python env or specify python path
```

Comment:
Since `one-cmds` script runs under the virtual env, I think it is hard to call local python. If you find the way, it would be good. If not, these seems to be possible solutions.

1. Get `requirement.txt` from user needed for running given script. And, install them with `pip` by `subprocess`. Maybe it needs to be done with `sudo` privilege. 
2. Get python executable path from user when the `one-cmds` is being installed. 
3. List the libraries frequently used for data processing and install them in advance.

Third option looks good to me as of now.

Reply:
> Since `one-cmds` script runs under the virtual env, I think it is hard to call local python.

I thought so, but when I tried to run `subprocess.run(['python', 'some_python_script.py'])` in venv, the local python is executed for the subprocess. It seems to run a subprocess with a new bash. 
The problem is how to make user specify the python path. 

Anyway, I prefer the third option, too. :smile: